test/addons/ocm-cluster/test

```diff
@@ -41,6 +41,7 @@ def wait_for_deployment(cluster, hub):
         "deploy/example-deployment",
         "--for=condition=available",
         "--timeout=120s",
+        "--namespace=default",
```

Comment:
We really should use a specific namespace for the tests (e.g. ocm-test like argocd-test) but this is good enough for now.